README.md

```diff
@@ -17,6 +17,10 @@
     rosdep-enabled: 'true'
     # Additional args to pass to rosdep install (optional, default: '-r')
     rosdep-install-args: '-r'
+    # Indicate if ROS PPA should be added (optional, default: 'false').
+    # The ROS PPA must be added if the Linux OS on which this action runs does not already contain a ROS distro.
+    # The ROS PPA should not be added if the Linux OS already contains a ROS distro (e.g., ROS Docker image)
+    add-ros-ppa: 'false'
```

Comment:
Shouldn't the default be true because it is needed for adding colcon?

Reply:
I use this action about half and half for ROS and non-ROS projects personally, so it doesn't really matter to me. I can set it to true so `tesseract` doesn't have to change if you want

Reply:
Lets do that. Thank you.

Reply:
Done